README.md

```diff
@@ -252,10 +252,6 @@ WMCO / Windows nodes does not work with the following products:
 * [OpenShift monitoring of user defined project](https://docs.openshift.com/container-platform/latest/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects)
 * [HugePages](https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/)
 
-### Accessing secure registries
```

Comment:
should we list private registry access in the "Enabled features" section?

Reply:
have mixed feelings about that, if so I would  join Mansi's statement  https://github.com/openshift/windows-machine-config-operator/pull/2168#issuecomment-2107810780